src/web-mercator-utils.js

```diff
@@ -242,7 +242,7 @@ export function getProjectionParameters({
 // PROJECTION MATRIX: PROJECTS FROM CAMERA (VIEW) SPACE TO CLIPSPACE
 // This is a "Mapbox" projection matrix - matches mapbox exactly if
 // nearZMultiplier === 1 / height
-// farZMultiplier === height
+// farZMultiplier === 1
```

Comment:
Comment on the fact that there are two versions of mapbox projection matrices to avoid someone losing time on this?

People use this stuff in ways we don't know about, thinking e.g. about the NY Times blog post, and not everyone upgrades to the latest version asap...